index.js

```diff
@@ -1,6 +1,37 @@
 const http = require('http');
+const mongoose = require('mongoose');
 require('dotenv').config();
 
+const { Schema } = mongoose;
+
+const movieScheme = new Schema({
+  title: String,
+  overview: String,
+  releaseDate: String,
+  genresIds: [{
```

Comment:
Let's use the same movie keys as in TMDM. So, instead of `genresIds` - `genre_ids`